pkgs/development/libraries/jemalloc/common.nix

```diff
@@ -1,6 +1,13 @@
-{ stdenv, fetchurl, version, sha256, ... }@args:
+{ version, sha256 }:
+{ stdenv, fetchurl,
+# By default, jemalloc puts a je_ prefix onto all its symbols on OSX, which
+# then stops downstream builds (mariadb in particular) from detecting it. This
+# option should remove the prefix and give us a working jemalloc.
+# Causes segfaults with some software (ex. rustc), but defaults to true for backward
+# compatibility. Ignored on non OSX.
+stripPrefix ? true }:
```

Comment:
Have you considered not ignoring it and setting `stripPrefix ? !stdenv.hostPlatform.isDarwin)` ?

Reply:
I have not considered that. I just wanted to keep the diff to the original commit as small as possible. It probably makes sense to do that change thought since the only user is really just rust.